docs/content/Guides/Data-Blending.md

```diff
@@ -7,6 +7,8 @@ subCategory: Tutorials
 menuOrder: 24
 ---
 
+[[info | ]] | This content is being moved to the [Cube.js community forum](https://forum.cube.dev/). | We encourage you to follow the content and discussions [in the new forum post](https://forum.cube.dev/t/data-blending-cube/).
+
```

Comment:
Hi @rpaik :wave:

Just thought I'd take this opportunity to show you how we've been doing info boxes. The `[[info | ]]` bit needs to be on its own line, then each line of text inside the box needs to be prefixed with `| `. We've also been wrapping the info boxes with the Prettier ignore start/end tags so we can format the documentation without breaking the info boxes (since they are nonstandard Markdown).

```suggestion
<!-- prettier-ignore-start -->
[[info | ]]
| This content is being moved to the [Cube.js community forum](https://forum.cube.dev/). 
| We encourage you to follow the content and discussions [in the new forum post](https://forum.cube.dev/t/data-blending-cube/).
<!-- prettier-ignore-end -->
```